src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java

```diff
@@ -345,11 +345,13 @@ public void updateAllocationNoteWithDetails(String caseUUIDString, String stageU
         String oldTeamName =  infoClient.getTeam(caseworkClient.getStageTeam(UUID.fromString(caseUUIDString), UUID.fromString(allocationStageUUID))).getDisplayName();
         String caseworkStageType = caseworkClient.getStageType(UUID.fromString(caseUUIDString), UUID.fromString(allocationStageUUID));
         StageTypeDto stageTypeDto = infoClient.getAllStageTypes().stream().filter(st -> st.getType().equals(caseworkStageType)).findFirst().get();
+        String allocationNoteText = allocationNote.isBlank() ? " stage" :" stage: " + allocationNote;
```

Comment:
Whilst this might solve the immediate issue... I don't think its a the right solution to the problem. I think you might be better off creating a new method in this class called `updateAllocationDetails` that constructs the string with the details only rather than if statement to manipulate the string in this way. It would be cleaner and more precise. You would need to alter the bpmn to call this new method. And write a test for your new method.